packages/test/test-end-to-end-tests/.npmrc

```diff
@@ -0,0 +1,10 @@
+@microsoft:registry=https://pkgs.dev.azure.com/fluidframework/internal/_packaging/office/npm/registry/
+always-auth=true
+
+# The @ff-internal packages only exist in our build feed.
+@ff-internal:registry=https://pkgs.dev.azure.com/fluidframework/internal/_packaging/build/npm/registry/
+always-auth=true
```

Comment:
Besides this, one needs to set up auth to that feed in the user-level .npmrc file.